site/hooks/useBalance.js

```diff
@@ -10,7 +10,8 @@ export const useBalance = function ({ symbol }) {
 
   const { data, mutate, error } = useSWR(
     [`${symbol}-Balance-${userAccount}`, userAccount, erc20],
-    async function (_, account, erc20Instancer) {
+    // eslint-disable-next-line no-unused-vars
+    async function ([_, account, erc20Instancer]) {
```

Comment:
now that this is an array, I think you can use destructuring and simply omit the first variable
 
```suggestion
    async function ([, account, erc20Instancer]) {
```

so there's no need anymore to disable the lint rule

Reply:
right, didn't consider that. here: https://github.com/hemilabs/pure.finance/pull/1/commits/df53fc81934898b30ca242efe2b83df564cf9ef7